paddle/utils/Logging.h

```diff
@@ -191,7 +191,7 @@ void installFailureWriter(void(*callback)(const char*, int));
 }
 #endif  // PADDLE_USE_GLOG
 
-#ifndef NDEBUG
+#ifdef NDEBUG
```

Comment:
It means DBG will be disabled when release. And when debug, it just like VLOG(5).

It may harm release performance to enable it in release.